### PR TITLE
README Final Touches

### PR DESCRIPTION
**Pull request by kj800x**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,9 +3,41 @@ AI agent to play the Wikipedia Game
 
 Final project for CS 4100 Foundations of Artificial Intelligence at Northeastern University
 
-This is a project to find a series of Wikipedia pages linking one to another in as few page loads as possible (not necessarily the shortest path). It uses an A* search and a variety of different heuristics are being tested. It will eventually have more advanced heuristics utilizing NLP and other classification methods.
+This is a project to find a series of Wikipedia pages linking one to another in as few page loads as possible (not necessarily the shortest path). It uses an A* search and a variety of different heuristics are being tested.
```
Comment (libreite) [inline README.md]: it can also use greedy

**Review by libreite — APPROVED** [review]

File: src/apis/WikipediaApi.py [hunk 2]
```diff
@@ -17,7 +17,7 @@
 class WikipediaApi(AWikiApi):
     def __init__(self):
         self.api_root = "https://en.wikipedia.org/api/rest_v1/"
-        self.headers = {'User-Agent': 'wong.mich@husky.neu.edu'}  # Wikipedia asks to provide contact info in user agent
+        self.headers = {'User-Agent': 'your@email.here'}  # Wikipedia asks to provide contact info in user agent
```
Comment (Emporophobe) [inline src/apis/WikipediaApi.py]: This should really be an environment variable or something but whatever

**Review by wugologist — APPROVED** [review]
LGTM

File: src/apis/WikipediaApi.py (hunk 2 above)
Comment (kj800x) [reply src/apis/WikipediaApi.py]: :man_shrugging:

File: README.md (hunk 1 above)
Comment (kj800x) [reply README.md]: Fine

**Pull request merged**